content/use-cases/tokenization/stablecoin-issuer.md

```diff
@@ -4,504 +4,173 @@ parent: tokenization.html
-- The amount of the transmittal order,
-- The execution date of the transmittal order, and
-- The identity of the recipient's financial institution.
+### Global Freeze
 
-<!-- SPELLING_IGNORE: transmittor -->
+If you see signs of suspicious activity, you can enact a global freeze on your account to prevent users from sending your tokens to each other and trading your token in the decentralized exchange. 
 
-See also:
+![Global Freeze](img/uc-stablecoin-global-freeze.png)
 
-- [Funds “Travel” Regulations: Questions & Answers ](https://www.fincen.gov/resources/statutes-regulations/guidance/funds-travel-regulations-questions-answers)
+See [Enact Global Freeze](enact-global-freeze.html)
 
-### Fee Disclosure and Tracing Funds
 
-- In the United States, Dodd Frank 1073 Electronic Fund Transfer Act (Regulation E) requires banks to provide information on cost and delivery terms for international payments originating in the US including exchange rate, fees, and the amount to be received by the designated recipient in the foreign country. "Pre-payment disclosure" is provided to a consumer when requesting an international electronic payment and “receipt disclosure” is provided to a consumer at the time consumer authorizes the transfer.
+### Clawback
 
-    See also:
+Clawback is an optional setting that you can choose before you begin to distribute your stablecoin. For regulatory purposes, some issuers _must_ have the ability to recover issued tokens after they are distributed to accounts. For example, if an issuer were to discover that tokens were sent to an account sanctioned for illegal activity, the issuer could recover, or _claw back_, the funds.
 
-    - [The Consumer Financial Protection Bureau description of the regulation and extensions for banks](https://www.consumerfinance.gov/rules-policy/final-rules/electronic-fund-transfers-regulation-e/#rule)
+See [Clawback](clawback.html).
 
-- In the European Union, EU Funds Transfer Regulation requires that the originator’s bank, the beneficiary’s bank, and any intermediary banks include certain details of the payer and payee in transaction details to detect, investigate, and prevent money laundering and terrorist financing.
+![Clawback](img/uc-stablecoin-clawback.png)
 
-    See also:
+### Burn
 
-    - [EU Regulation (EC) No 1781/2006 description](http://eur-lex.europa.eu/LexUriServ/LexUriServ.do?uri=OJ:L:2006:345:0001:0009:EN:PDF)
+One way to manage the value of a stablecoin is to destroy, or _burn_ stablecoins. When you burn stablecoins, you reduce the number of tokens in circulation. That will typically increase the value of the remaining tokens as supply falls.
 
-    - [Effective June 26, 2017: Regulation 2015/847 on information accompanying transfers of funds](http://eur-lex.europa.eu/legal-content/EN/ALL/?uri=CELEX%3A32015R0847)
+To destroy stablecoins, you can transfer them to a "black hole" account (an account address where no one knows the secret key).
```

Comment:
This is misleading/incorrect. You cannot always transfer tokens to a black hole account (it would have to have created a trust line to your issuer before becoming a black hole). However, you _can_ always send tokens back to the issuing account, which directly reduces the number of tokens outstanding. That's burning the tokens in a more direct and literal sense.

However, there's something off about the bigger picture here. For a non-stablecoin token with a fixed total supply  and a blackholed issuer, it might make sense to talk about "burning" tokens. But for a stablecoin, it's weird; typically, the same process is called "redeeming" the tokens (for their off-ledger collateral value). The way a stablecoin (specifically) is supposed to work, the on-ledger supply fluctuates in response to increases or decreases in collateral (that is, deposits and withdrawals) and tokens are "burned" or "issued" regularly.